src/api/action.ts

```diff
@@ -26,7 +26,7 @@ export const getActions = async (
 
 export const getMemberActions = async (
   { API_HOST, axios }: PartialQueryConfigForApi,
-  args: { startDate?: string; endDate?: string } = {},
+  args: { startDate: string; endDate: string },
```

Comment:
Why the change? did you check that we want to enforce?

Reply:
It was useless to allow undefined values based on the usage of the hook. I prefer to be tight about the values of the arguments. Also it reduced the complexity in other parts of the code.